src/components/Form/TextInput.tsx

```diff
@@ -94,6 +96,14 @@ export class BaseTextInput extends Component<TextInputProps> {
   }
 }
 
+export const StyledBaseTextInput = styled((props: TextInputProps) => (
```

Comment:
Should these styles be here or simply in the TextInput/SearchInput basestyles? Are there even cases where one would want a disabled search input on the screen?

Reply:
I was wondering the same. In addition, I'd avoid exporting something named with arbitrary Styled-prefix, because the styled does not have any particular meaning outside this context (why styled?).

Reply:
> Should these styles be here or simply in the TextInput/SearchInput basestyles? 

Maybe they can be moved there if it possible; might need to change then also how it is done with the Button's one to keeping it constant. Currently by this way, one advantage is that there is no need to tell 'disabledClassName' again in the baseStyles (DRY).

---

> Are there even cases where one would want a disabled search input on the screen?

Maybe there is no use for disabled search, that is true. Maybe we should then remove this from the SearchInput.

---

>.. I'd avoid exporting something named with arbitrary Styled-prefix, because the styled does not have any particular meaning outside this context (why styled?).

**Currently** exporting it out with styled-prefix to keep consistent with other places when using styled-components and because it is used in the `SearchInput.tsx` also.

Reply:
> Maybe there is no use for disabled search, that is true. Maybe we should then remove this from the SearchInput.
We cannot know whether there is usecase for this or not, so I'd keep the functionality.

> Currently exporting it out with styled-prefix to keep consistent with other places when using styled-components and because it is used in the SearchInput.tsx also.
Let's keep it consistent then. Are these also exported externally meaning not only within the library? Because like mentioned, word ```Styled``` does not make any sense in the context of library API... TwoHardThings?

Reply:
> Are these also exported externally meaning not only within the library? Because like mentioned, word Styled does not make any sense in the context of library API... TwoHardThings?

Seems that it is not exported in any of the indexs'